snap7/client.py

```diff
@@ -616,6 +616,54 @@ def set_plc_datetime(self, dt):
 
         return self._library.Cli_SetPlcDateTime(self._pointer, byref(buffer))
 
+    def as_read_area(self, area, dbnumber, start, size):
```

Comment:
i think i added a as_read_area() also in my cleanup (with a raise NotImplemented exception)

Reply:
There is as_read_area and asreadarea. So you want them with a NotImplemented exception?

Reply:
oops, o forgot to make those methods `as_read_area` format, my bad. No, I made all those NotImplemented methods as a reminder of which methods still need to be implemented, you can remove that one if you add it now.

Reply:
As well, how would you like the names to be dealed? The already implemented methods have underscores (as_ab_read) and some not (asebread). I prefere the underscore version for these kind of methods, because it is easier to read. Other methods like the `async_wait_loop()` could be without underscores and are still readable.

Reply:
underscore version is preferred.